src/components/pages/ViewAllMeetings/MeetingInfo.js

```diff
@@ -1,10 +1,15 @@
 import React from 'react';
+import './MeetingInfo.css';
 
 const MeetingInfo = props => {
   const { info } = props;
 
+  console.log(props);
```

Comment:
We often use these console logs until we are production-ready, so I guess we are just keeping an eye on these at this point.